src/trufflecon2020.html

```diff
@@ -102,16 +114,16 @@ <h2>Grow with the Blockchain Community</h2>
 
 </main>
 
-<main class="banner banner-tcon-2020 pt-5 pb-5">
+<main class="banner-tcon-2020 pt-5 pb-5">
   <div class="container">
     <div class="row">
       <div class="col-md-12">
-          <!--<nav class="nav trufflecon-nav-inline justify-content-center">
-            <a class="nav-link scroll-link" href="#speakers">2020 Speakers</a>
-            <a class="nav-link scroll-link" href="#sponsors">2020 Sponsors</a> 
-          </nav>-->
           <h3 class="mt-6 mb-6 mx-auto text-center" style="width: 60%; letter-spacing: 1px;">A conference for those who want to build world-changing applications powered by decentralized technologies.</h3>
           <h1 class="tcon-countdown color-drizzle text-center mb-4 text-lowercase"></h1>
+          <nav class="nav trufflecon-nav-inline justify-content-center">
+            <a class="nav-link scroll-link" href="#speakers">2020 Speakers</a>
+            <a class="nav-link scroll-link" href="#sponsors">2020 Sponsors</a> 
```

Comment:
Having `2020` here feels a bit redundant since we're no longer showing 2019 speakers/sponsors

Reply:
Agreed